apps/crn-server/test/fixtures/reminders.fixtures.ts

```diff
@@ -375,3 +379,108 @@ export const getSquidexReminderEventsContents = (): NonNullable<
     },
   };
 };
+
+export const getContentfulRemindersGraphqlResponse =
+  (): FetchRemindersQuery => ({
+    eventsCollection: {
+      items: [getContentfulReminderEventsCollectionItem()],
+    },
```

Comment:
hmm I've ended up never using this fixture, in the tests since I change either the content of the events or users I've ended up writing this in the test itself. I think maybe it's better to remove `getContentfulRemindersGraphqlResponse` then

Reply:
solved here [remove fixture getContentfulRemindersGraphqlResponse](https://github.com/yldio/asap-hub/pull/3236/commits/d055b1047af5eb0c18cee780666d562ce448d84b)